08LessonEight/in-class-project/css/homework-style.css

```diff
@@ -0,0 +1,88 @@
+.clearfix:after{
+    visibility: hidden;
+    display: block;
+    font-size: 0;
+    content: " ";
+    clear: both;
+    height: 0;
+}
+.recentblog{
+  text-align: center;
+}
+.hands{
+  background-color:tan;
+  width:350px;
+  height:300px;
+  padding-left:20px;
+  padding-top:20px;
+  padding-right:20px;
+  padding-bottom:20px;
+  float: left;
+  margin-right:20px;
+}
+.small1{
+  background-color:tan;
+  width:320px;
+  height:90px;
+  padding-left:15px;
+  padding-top:15px;
+  padding-right:15px;
+  padding-bottom:15px;
+  float:left;
+  margin-bottom:15px;
+}
+.small2{
+  background-color:tan;
+  width:320px;
+  height:90px;
+  padding-left:15px;
+  padding-top:20px;
+  padding-right:15px;
+  padding-bottom:15px;
+  float: left;
+  margin-bottom:15px;
+}
+.small3{
+  background-color:tan;
+  width:320px;
+  height:90px;
+  padding-left:15px;
+  padding-top:15px;
+  padding-right:15px;
+  padding-bottom:15px;
+  float: left;
+  margin-bottom:15px;
+}
+.h3{
+  page-break-after:auto;
+}
+.clear{
+  clear:both;
+}
+.two{
+  margin-top: auto;
+}
+.list1{
+  float:left;
+  margin-top: 50px;
+  padding-left:15px;
+  padding-top: 15px;
+  padding-right:15px;
+  padding-bottom:15px;
+}
+.list2{
+  float:left;
+  margin-top: 50px;
+  padding-left:15px;
+  padding-top: 15px;
+  padding-right:15px;
+  padding-bottom:15px;
+}
+.list3{
+  float:left;
+  margin-top: 50px;
+  padding-left:15px;
+  padding-top: 15px;
+  padding-right:15px;
+  padding-bottom:15px;
+}
```

Comment:
there are a LOT of similarities between the classes you've declared; can you combine then to simplify your code?